### DISCUSSION TITLE
Options to Simplify

### DISCUSSION THREAD

**Discussion by tannerphilp** [kinfoundation/kin-rfcs#123] [_ Archive - KRE 3.1 Discussion]
### Logic
This proposal has a lot of good content. I like the push to incentivize the growth of the participation in the economy. 

The one challenge I could see is that this does introduce some complexity for developers to understand and optimize for the logic. I'm wondering if there might be an option to simplify this and get the same desired outcome. 

A lens I might suggest would be that a developer could look at the input data to the KRE, easily understand where the opportunity is, and forecast how they could benefit.

A potential option would be: KRE Score = Median AUB * AU

If I was a prospective developer in the ecosystem I could then look at this and do a quick projection of: 
- Assume **X%** of my MAU will have a balance
- Assume the median balance of those users would be **Y**

I know that the ecosystem currently has: 
- **X** Active Users
- **Y** Median AUB

Therefore my portion of the daily payout would be: App(X*Y) / Ecosystem(X*Y) 

### Payout
I like that there is a push to recalibrate the payout to align with USD value. There has been a lot of volatility over the last year. This proposal is for the new cap to go in on June 1, which is 120 days from this proposal. If we look back 120 days the price of Kin was ~2x. 

Would it make sense to say that on June 1 the cap will be set based on a USD value applied to a median value from a preceding period of time, to try and get it closest to the intended outcome? 

For example, starting cap on June 1 is $5,000 USD / Median price of Kin over the last 30 days. 

///

These are two thoughts on driving to simplicity. Interested to hear others's thoughts.

**Comment by showered** [comment]
**Logic** - I think you've pretty much described the existing KRE?

**Payout** - I personally don't like the idea of capping KRE payments to a USD value for two reasons:

1. As more apps join the ecosystem, KRE will continue to become less and less profitable
2. There becomes an incentive for developers to drive the price of Kin down. Or at the very least, there is no longer a benefit to developers when the price goes up.

**Comment by skyf0xx** [comment]
@tannerphilp you raised two issues, could you clarify aligning the payout with the USD value?

How would you comment on these previous proposals looking for a similar effect? (In that the environment has changed since the previous proposal)
1. [Capping based on liquidity](https://github.com/kinecosystem/rewards-engine/blob/master/accepted/kik_kre_3_0_cap.md#liquidity-cap)
2. [Capping based on USD](https://github.com/kinecosystem/rewards-engine/blob/master/accepted/kik_kre_3_0.md#daily-payout-cap)
